research/src/pages/focusgroup/focusgroup.explainer.mdx

```diff
@@ -361,48 +361,50 @@ When extending a focusgroup, traversal order is based on document order. Given t
 Example 6:
 
 ```html
-<div focusgroup>
-  <div id="A" tabindex="0"></div>
-  <div id="B" tabindex="-1">
-    <div>
-      <div focusgroup="extend">
-        <div id="B1" tabindex="-1"></div>
-        <div id="B2" tabindex="-1"></div>
+<div role=toolbar focusgroup>
+  <button id="A" type=button tabindex="0"></button>
+  <button id="B" type=button tabindex="-1"></button>
+  <div>
+      <div role=radiogroup focusgroup="extend">
+        <button id="B1" role=radio type=button tabindex="-1"></button>
+        <button id="B2" role=radio type=button tabindex="-1"></button>
       </div>
-    </div>
   </div>
-  <div id="C" tabindex="-1"></div>
+  <button id="C" type=button tabindex="-1"></button>
 </div>
 ```
 
 Sequentially pressing the right arrow (assuming `horizontal-tb` + LTR writing-mode) would move
-through the `<div>`s: A, B, B1, B2, C. (And in reverse direction: C, B2, B1, B, A, as expected.)
+through the `<button>`s: A, B, B1, B2, C. (And in reverse direction: C, B2, B1, B, A, as expected.)
 
 When the extending linear focusgroup is axis-aligned (see following sections), the wrapping state
 of an ancestor focusgroup definition is applied to it. For example: specifying `wrap` on an
-extending focusgroup definition when "nowrap" (even implicitly) was specified on a parent
-focusgroup (of the same supported axis) does not make sense; the extending focusgroup candidates
+extending focusgroup definition (the radiogroup) when "nowrap" (even implicitly) was specified on a parent
+focusgroup (the toolbar; with the same supported axis) does not make sense; the extending focusgroup candidates
 join the rest of the ancestor focusgroup candidates to form one logical ordering--they do not
 define a separate range of focusgroup candidates to apply different wrapping logic onto.
 
 Example 7:
 
 ```html
 <!-- This is an example of what NOT TO DO -->
-<div focusgroup>
-  <div id="A" tabindex="0"></div>
-  <div id="B" tabindex="-1" focusgroup="extend wrap">
-    <!-- 'wrap' will not apply -->
-    <div id="B1" tabindex="-1"></div>
-    <div id="B2" tabindex="-1"></div>
+<div role=toolbar focusgroup>
+  <button id="A" type=button tabindex="0"></button>
```

Comment:
Does specifying the `type=button` attribute bring something valuable to the example? Since we're not inside of a form here, the button won't have any action, right?

Disclaimer: I am not an expert on the button type attribute and might be missing something.

Reply:
I'm pretty sure that's standard practice to make it a simple push-button, otherwise it ends up looking like a submit button without anything to submit... which is weird. (Honestly, I think HTML got it wrong by having submit button be the default, but can't change that now.)